build is failing:
```
======================================================================
ERROR: test_report_runner (custom.ilsgateway.tests.test_report_runner.TestReportRunner)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/ilsgateway/tests/test_report_runner.py", line 105, in test_report_runner
    report_run(TEST_DOMAIN)
  File "/vendor/lib/python2.7/site-packages/celery/app/task.py", line 420, in __call__
    return self.run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/decorators.py", line 122, in _inner
    return fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/custom/ilsgateway/tasks.py", line 73, in report_run
    populate_report_data(run.start, run.end, domain, run, strict=strict)
  File "/mnt/commcare-hq-ro/custom/ilsgateway/tanzania/warehouse/updater.py", line 255, in populate_report_data
    for org in chunk
  File "/vendor/lib/python2.7/site-packages/celery/canvas.py", line 362, in __call__
    return self.apply_async(args, kwargs)
  File "/vendor/lib/python2.7/site-packages/celery/canvas.py", line 251, in apply_async
    return _apply(args, kwargs, **options)
  File "/vendor/lib/python2.7/site-packages/celery/app/builtins.py", line 275, in apply_async
    return self.apply(args, kwargs, **options)
  File "/vendor/lib/python2.7/site-packages/celery/app/builtins.py", line 302, in apply
    last and (last.get(), ),
  File "/vendor/lib/python2.7/site-packages/celery/canvas.py", line 181, in apply
    return self.type.apply(args, kwargs, **options)
  File "/vendor/lib/python2.7/site-packages/celery/app/task.py", line 743, in apply
    request=request, propagate=throw)
  File "/vendor/lib/python2.7/site-packages/celery/app/trace.py", line 355, in eager_trace_task
    uuid, args, kwargs, request)
  File "/vendor/lib/python2.7/site-packages/celery/app/trace.py", line 253, in trace_task
    I, R, state, retval = on_error(task_request, exc, uuid)
  File "/vendor/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/mnt/commcare-hq-ro/custom/ilsgateway/tanzania/warehouse/updater.py", line 493, in process_non_facility_warehouse_data
    % (location.name, str(location.location_id), len(facs)))
TypeError: object of type 'itertools.imap' has no len()

======================================================================
ERROR: test_can_clear_locations (corehq.apps.commtrack.tests.test_locations.MultiLocationsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_locations.py", line 210, in test_can_clear_locations
    self.assertEqual(len(user.locations), 0)
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1702, in locations
    return list(SQLLocation.objects
NameError: global name 'SQLLocation' is not defined
```

--- a/corehq/apps/users/models.py
+++ b/corehq/apps/users/models.py
@@ -1684,6 +1684,7 @@ class CommCareUser(CouchUser, SingleMembershipMixin, CommCareMobileContactMixin)
         locations per user flag set. It will error if you try
         to call it on a normal domain.
         """
+        from corehq.apps.locations.models import SQLLocation
         if not self.project.supports_multiple_locations_per_user:
             raise InvalidLocationConfig(
                 "Attempting to access multiple locations for a user in a domain that does not support this."

--- a/custom/ilsgateway/tanzania/warehouse/updater.py
+++ b/custom/ilsgateway/tanzania/warehouse/updater.py
@@ -474,9 +474,11 @@ def process_facility_transactions(facility_id, transactions, start_date, end_dat
 
 
 def get_non_archived_facilities_below(location):
-    return location.sql_location.get_descendants(include_self=True).filter(
-        is_archived=False, location_type__name='FACILITY'
-    ).values_list('location_id', flat=True).couch_locations()
+    return list(location.sql_location
+                .get_descendants(include_self=True)
+                .filter(is_archived=False,
+                        location_type__name='FACILITY')
+                .couch_locations())
 
 
 @task(queue='logistics_background_queue')

